src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let formattedDate = date.toString();
```

Comment:
Instead of immediately formatting the `date` input, I'm converting it to a string. The reason being is because calling `.indexOf()` on anything other than a string will throw an error, preventing the app from even rendering. While the changes of this happening are small, I wanted to at least convert the date to a string first so that the entire app won't fail to load if this happens.